tests/cfg_test.py

```diff
@@ -215,35 +218,117 @@ def test_orelse(self):
+            self.connected(function_entry, ret_of_subprocess_call),
+            self.connected(ret_of_subprocess_call, ret_does_this_kill_us_equal_call_5),
+            self.connected(ret_does_this_kill_us_equal_call_5, function_exit),
+            self.connected(function_exit, restore_node),
+            self.connected(restore_node, return_handler),
+            self.connected(return_handler, print_so),
 
-                          self.connected(save_node, assign_to_temp),
-                          self.connected(assign_to_temp, assign_from_temp),
-                          self.connected(assign_from_temp, function_entry),
-                          self.connected(function_entry, ret_of_subprocess_call),
-                          self.connected(ret_of_subprocess_call, ret_does_this_kill_us_equal_call_5),
-                          self.connected(ret_does_this_kill_us_equal_call_5, function_exit),
-                          self.connected(function_exit, restore_node),
-                          self.connected(restore_node, return_handler),
-                          self.connected(return_handler, print_so),
+            self.connected(print_so, print_good),
+            self.connected(print_good, _exit)
+        ])
 
-                          self.connected(print_so, print_good),
-                          self.connected(print_good, _exit)])
+    def test_orelse_with_no_variables_to_save(self):
+        self.cfg_create_from_file('examples/example_inputs/try_orelse_with_no_variables_to_save.py')
+
+        self.nodes = self.cfg_list_to_dict(self.cfg.nodes)
+        self.assert_length(self.cfg.nodes, expected_length=15)
+
+        entry = 0
+        try_ = 1
+        print_a5 = 2
+        except_im = 3
+        print_wagyu = 4
+        temp_3_diff = 5
+        diff = 6
+        function_entry = 7
+        ret_subprocess_call = 8
+        ret_does_this_kill_us_4 = 9
+        exit_does_this_kill_us = 10
+        ret_does_this_kill_us_3 = 11
+        print_so = 12
+        print_good = 13
+        _exit = 14
+
+        self.assertInCfg([
+            self.connected(entry, try_),
+            self.connected(try_, print_a5),
+            self.connected(print_a5, except_im),
+            self.connected(print_a5, temp_3_diff),
+            self.connected(print_a5, print_good),
+            self.connected(except_im, print_wagyu),
+            self.connected(print_wagyu, print_good),
+            self.connected(temp_3_diff, diff),
+            self.connected(diff, function_entry),
+            self.connected(function_entry, ret_subprocess_call),
+            self.connected(ret_subprocess_call, ret_does_this_kill_us_4),
+            self.connected(ret_does_this_kill_us_4, exit_does_this_kill_us),
+            self.connected(exit_does_this_kill_us, ret_does_this_kill_us_3),
+            self.connected(ret_does_this_kill_us_3, print_so),
+            self.connected(print_so, print_good),
+            self.connected(print_good, _exit),
+
```

Comment:
You can delete this line and dedent the `])` to be on the same level as `self.assertInCfg([`.